libs/core/src/lib/action-sheet/action-sheet-body/action-sheet-body.component.ts

```diff
@@ -73,5 +89,17 @@ export class ActionSheetBodyComponent {
         if (this._keyboardSupportService.keyManager) {
             this._keyboardSupportService.onKeyDown(event);
         }
+
+        if (event.shiftKey && KeyUtil.isKeyCode(event, TAB)) {
```

Comment:
KeyboardSupportService supports wrapping the keyboard navigation (cycling). You can try to adjust `this._keyboardSupportService.setKeyboardService(this.actionSheetItems, false);` and pass `true` to the second argument in `ActionSheetComponent`